### PR TITLE
Update Remote Grid execution configuration

### PR DESCRIPTION
**Pull request by DerekJLeggett**
The "DesiredCapabilities" object is crucial to configuring remote execution on the Selenium Grid. Updated so this is passed from the program using Crawljax as a library.

### CODE REVIEW

File: examples/src/main/java/com/crawljax/examples/RemoteExample.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+public class RemoteExample {
+
+    private static final long WAIT_TIME_AFTER_EVENT = 200;
+    private static final long WAIT_TIME_AFTER_RELOAD = 200;
+
+    /**
+     * Run this method to start the crawl.
+     */
+    public static void main(String[] args) {
+        String remoteUrl = "http://localhost:4444/wd/hub";
```
Comment (amesbah) [inline examples/src/main/java/com/crawljax/examples/RemoteExample.java:25]: Can you add comments on how to start this service locally?

File: examples/src/main/java/com/crawljax/examples/RemoteExample.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+        int numberOfBrowsers = 1;
+        CrawljaxConfigurationBuilder builder = CrawljaxConfiguration.builderFor("http://demo.crawljax.com/");
+        builder.crawlRules().setFormFillMode(FormFillMode.NORMAL);
```
Comment (amesbah) [inline examples/src/main/java/com/crawljax/examples/RemoteExample.java:28]: demo.crawljax.com does not exist anymore. Can you replace it with a working link? The example should ideally be runnable.

**Review by amesbah — COMMENTED** [review]
Looks good overall. See my two comments below on the example.

File: examples/src/main/java/com/crawljax/examples/RemoteExample.java (hunk 2 above)
Comment (DerekJLeggett) [reply examples/src/main/java/com/crawljax/examples/RemoteExample.java:32]: I set to a working URL.

File: examples/src/main/java/com/crawljax/examples/RemoteExample.java (hunk 1 above)
Comment (DerekJLeggett) [reply examples/src/main/java/com/crawljax/examples/RemoteExample.java:29]: I added comments with links to help get started.

**Review by amesbah — APPROVED** [review]

**Comment by amesbah** [comment]
Resolves #528

**Pull request merged**